ember_debug/object-inspector.js

```diff
@@ -22,6 +22,39 @@ try {
 
 const keys = Object.keys || Ember.keys;
 
+/**
+ * Add Known Ember Mixins and Classes so we can label them correctly in the inspector
+ */
+const emberNames = new Map();
```

Comment:
NOTE: this is a nitpick but you can instantiate a [`Map`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) directly from an array. I find it clearer:
```js
const emberNames = new Map([
  [Ember.Evented, 'Evented Mixin']
]);
```